### PR TITLE
feat(sass-true): add types for v6.0

### PR DESCRIPTION
**Pull request by robertmaier**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/sass-true/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+export interface Assertion {
+    description: string;
+    output: string;
+    assertionType: string;
+    expected: string;
+    details?: string;
+}
+
+export interface Module {
```
Comment (amcasey) [inline types/sass-true/index.d.ts:21]: Where are `Module` and `Test` used?

**Review by amcasey — CHANGES_REQUESTED** [review]

File: types/sass-true/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    module: string;
+    tests: Test[];
+}
+
+export interface Test {
+    assertions: Assertion[];
+}
+
+export interface Assertion {
```
[the remark this answers is not in the document]
Comment (robertmaier) [reply types/sass-true/index.d.ts]: my bad, i removed the duplicate definition

File: types/sass-true/index.d.ts (hunk 1 above)
Comment (robertmaier) [reply types/sass-true/index.d.ts]: `Module` is used as the return type of the `parse` function. Test is used within the `Module` type

Comment (amcasey) [reply types/sass-true/index.d.ts]: 🤦‍♂️

**Review by amcasey — APPROVED** [review]
Thanks!

**Comment by robertmaier** [comment]
Ready to merge

**Comment by robertmaier** [comment]
Thanks for the quick review @amcasey 👍

**Pull request merged**